src/test/fixtures/contents/testbook1-shortid:testpage1-shortid.json

```diff
@@ -4,6 +4,7 @@
   "title": "Test Page 1",
   "version": "1.0",
   "revised": "2018-07-30T15:58:45Z",
+  "book_state": "live",
```

Comment:
Why is this here? It's later imported in `archiveLoader` and typed as `ArchivePage` which doesn't have `book_state`

Reply:
nah, I've tried different things and didn't realize that this is response for archive pages. I've removed this